Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,10 +1,170 @@
+  * Added option to cancel the game after clicking **Ready**. 
+  * Added a gear icon for accessing and modifying app settings. 
+  * The UI now tracks and displays player has arrived, number of connected players in the lobby, and player status. 
+
+* Added F/X and animation assets for the game including:
+
+  * Audio files for boss sound effects
+  * Ramp-up animation for hero movement and actions
+  * Visual effects and particles for Tank charge skill 
+  * Art assets to wave spawner, including animations for ReceiveDamage, Broken (died), and Revive 
+  * Game music 
+
+* Added audiomixer to Boss Room for separate audio channels and a master mixer. 
+
+* Updated and added various hero abilities:
+
+  * Added a cooldown to Archer's PowerShot. 
+  * Added the Rogue's Dagger and Dash skills. 
+  * Added the Rogue's Sneak skill using local stealth, applying a graphical effect to the stealthy character while still sending all network events normally.
+  * Properly display Heal abilities when targeting a fallen ally character. 
+  * Character attack actions properly support Hold to charge options. 
+
+* To show how UI elements and game objects can be networked, added networked functionality using `INetworkSerializable` in the `CharSelect` screen to network player's selected character on the Character Selection screen. 
+* Added a filter to remove any invisible characters introduced by Zoom when copy-pasting room names, IP addresses, or ports. 
+* Boss Room now uses the [UnityToonShader](https://github.com/IronWarrior/UnityToonShader) for rendering the 3D surfaces to emulate 2D, flat surfaces.
+* Added disconnection error message to load when a player or host disconnects due to limited or no network connectivity. Client logic was also updated to detect Host disconnection scenarios, such as losting connectivity.
+* Balanced hero and enemy stats, spawners, trigger areas, and enemy detetction areas. 
+* Added healthbars to display when damaged and properly track imp health locally and across clients. 
+
+### Changes
+
+* Updated Boss Room per asset and project settings for increased performance and better resource management:
+
+  * Disabled GPU Skinning to optimize GPU usage and FPS.
+  * Lowered quality of ambient occlusion from high to medium.
+  * Switched SMAA High to FXAA (fast mode) reducing GPU cost.
+  * Modified GPU Instancing on imps, heroes, and the boss to significantly reduce the number of draw calls.
+  * Turned off Cast Shadows on Imp and Imp Boss.
+  * Disabled mesh colliders of lava, which is more decorative than interactive.
+  * Refactored the S_SimpleDissolve shader which consumed most import time.
+
+* Updated the Photon Setup Guide, indicating you need only app ID when playing with friends. For users connecting across regions, you may need to hard code a region in your app settings by using the room code and region instead of just the room code sharing in game. 
```

Comment:
mmm where is that coming from? Right now, regions are hardcoded in the room name you're sharing

Reply:
Hmm this was a ticket GOMPS-88, can remove this one if we want!